### ISSUE TITLE
Google docs seem too wide open

### ISSUE THREAD

**Issue by raravena80** [cncf/toc#395]
Any anonymous hacker can type anything on a bunch of TOC-SIG Google doc. I can basically type 'Anonymous was here' from an incognito window on these Google docs (and more):

- SIG-Security notes (Ironically 😄):
  - https://docs.google.com/document/d/170y5biX9k95hYRwprITprG6Mc9xD5glVn-4mB2Jmi2g
- SIG-Runtime notes:
  - https://docs.google.com/document/d/1k7VNetgbuDNyIs_87GLQRH2W5SLgjgOhB6pDyv89MYk
- SIG-Network notes:
  - https://docs.google.com/document/d/18hYemFKK_PC_KbT_TDBUgb0rknOuIhikkRxer4_bv4Q
- etc

I'm wondering if there's a better way to 
- Keep track of users who make the changes
- Allow certain people who request access
- Allow anonymous to comment only, etc.

cc: @amye @quinton-hoole

**Comment by duglin** [comment]
I believe some folks require people to be a member of a certain google group before they can add comments or even see docs. So that’s one option.

**Comment by amye** [comment]
There's version history if we need to use it, but it's not yet been an issue. 
Do you have some other tool in mind?

**Comment by caniszczyk** [comment]
By default, all docs should be comment only and edit access is given to who should have it.

I have never seen a gdoc hijacked yet in a CNCF related project so I think this is a fairly minor issue.

**Comment by raravena80** [comment]
Yes, I've never seen any of the CNCF docs hijacked either. Raising the issue for awareness.

- Google Groups is good because it's tightly integrated with gdocs. 
- Groups.io doesn't seem to have integration with gdocs(?)
- An alternative would be [Slack -> gdocs](https://gsuite.google.com/slack/) integration.
